index.mjs

```diff
@@ -0,0 +1,130 @@
+//importing game functions and images from resources folder
+import { ask, inputSanitizer, isInteger, random } from './resources/usableFunctions.mjs'
+import { title, win } from './resources/ascii_images.mjs'
+
+start();//starts the game
+
+async function start() {
+    //Initializes game with title, game choice, and range choice
+    console.log(title);
+    console.log(`Hey! Wanna play a guessing game?`);
+    let gameChoice = await ask(`Press (1) if you want me to guess a number you think of.\nPress (2) if you want to guess a number I think of.\n`);
+    //ensures player enters correct game option
+    while(gameChoice !== '1' && gameChoice !== '2') {
+        gameChoice = await ask(`Please answer (1) or (2)...\n`);
+    }
+    //determines max range to play
+    let maxRange = await ask(`Awesome! Let's try to guess a number between 1 and ....\n(enter a number, try 100, or even 1000 if you want it to be hard!)\n`);
+    
+    //ensures valid integer on input
+    maxRange = isInteger(maxRange, false);
+    while(maxRange === false) {
+        maxRange = await ask(`Please enter a whole, positive integer...\n`);
+        maxRange = isInteger(maxRange, false);
+    }
+    console.log(`Awesome! We are guessing a number between 1 and ${maxRange}!\n\nLet's Play Guess What!!!\n`);
+    
+    //Global variable declaration
+    let compGuess;
+    let playerGuess;
+    let guessCount = 1;
+    let guessArray = [];
```

Comment:
This are actually local variables to the start function, but since your compGame and playerGame functions are defined inside of your start function, they are able to see these variables and manipulate them

Reply:
That's an old comment written before I added the playAgain function that I forgot to change.  The start was added back in to make playAgain() functional.